docs/docs/92-development/03-ui.md

```diff
@@ -29,3 +29,10 @@ The following list contains some tools and frameworks used by the Woodpecker UI.
 - [eslint](https://eslint.org/)
 - [Volar & vue-tsc](https://github.com/johnsoncodehk/volar/) for type-checking in .vue file
   - use the take-over mode of Volar as described by [this guide](https://github.com/johnsoncodehk/volar/discussions/471)
+
+## Messages and Translations
+
+Woodpecker uses [Vue I18n](https://vue-i18n.intlify.dev/) as translation library. New message keys
+must be added at least to `web/src/assets/locales/en.json` otherwise they will be removed by weblate (a transaltion service) and everyone expects to translate from English to other languages
```

Comment:
```suggestion
Woodpecker uses [Vue I18n](https://vue-i18n.intlify.dev/) as translation library.  New translations have to be added to `web/src/assets/locales/en.json`. The English source file will be automatically imported into [Weblate](https://translate.woodpecker-ci.org/) (the translation system used by Woodpecker) where all other languages will be translated by the community based on the English source.
```

Reply:
Took that sentence but added a word to not provide any other translations. The mechanism was not clear to me.